bin/fetch-configlet

```diff
@@ -62,11 +67,11 @@ def fetch_configlet():
         headers = err.headers
         response = err.reason
     except URLError as err:
-        print(err.reason)
+        logger.error(err.reason)
```

Comment:
For debug purposes might be good to do `logger.exception(err)` so we get some traceback here.